os/net/socket/bsd_socket_api.c

```diff
@@ -277,41 +281,41 @@ in_addr_t inet_addr(FAR const char *cp)
 
 int inet_aton(const char *cp, struct in_addr *inp)
 {
-	return ip4addr_aton(cp, (ip4_addr_t*)inp);
+	return ip4addr_aton(cp, (ip4_addr_t *)inp);
 }
 
 FAR const char *inet_ntop(int af, FAR const void *src, FAR char *dest, socklen_t size)
 {
-
 #ifdef CONFIG_NET_IPv4
-	if (af == AF_INET)
-		return ip4addr_ntoa_r((const ip4_addr_t*)(src), ((char *)(dest)), (size));
-	else
+	if (af == AF_INET) {
+		return ip4addr_ntoa_r((const ip4_addr_t *)(src), ((char *)(dest)), (size));
+	} else
 #endif
 #ifdef CONFIG_NET_IPv6
-	if (af == AF_INET6)
-		return ip6addr_ntoa_r((const ip6_addr_t*)(src), ((char *)(dest)), (size));
-	else
+		if (af == AF_INET6) {
```

Comment:
You should remove a tab newly added for these four lines.
A tab is right. not two tabs